docs/modules/configuring/con-kafkaproxy-resource-allocation.adoc

```diff
@@ -0,0 +1,68 @@
+// file included in the following:
+//
+// kroxylicious-operator/assemblies/assembly-operator-operate-resource-allocation.adoc
+
+[id='con-kafkaproxy-tuning-resource-allocation-{context}']
+== Configuring Proxy container CPU and memory resource limits and requests
+
+When you define a `KafkaProxy` resource, a Kubernetes Deployment of Proxy containers is created. Each of these containers runs a single Kroxylicious Proxy process.
+
+By default, these Proxy containers are configured with the following resources:
+
+.Default Proxy container resources
+[source,yaml]
+resources:
+  requests:
+    cpu: '500m'
+    memory: '512Mi'
+  limits:
+    cpu: '500m'
+    memory: '512Mi'
+----
+
+You can learn more about Kubernetes resource management https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[here].
+
+To replace these default resource allocations, modify the `proxyContainer` section within your `KafkaProxy` specification.
+
+.Example `KafkaProxy` configuration with proxy container resource specification
+[source,yaml]
+----
+kind: KafkaProxy
+apiVersion: kroxylicious.io/v1alpha1
+metadata:
+  namespace: my-proxy
+  name: simple
+spec:
+  infrastructure:
+    proxyContainer:
+      resources:
+        requests:
+          cpu: '400m'
+          memory: '656Mi'
+        limits:
+          cpu: '500m'
+          memory: '756Mi'
+----
+
+Note that the `resources` from the KafkaProxy resource will entirely replace the defaults. So you can do things like remove the
+CPU requests/limits:
+
+.Example `KafkaProxy` configuration removing cpu requests/limits
+[source,yaml]
+----
+kind: KafkaProxy
+apiVersion: kroxylicious.io/v1alpha1
+metadata:
+  namespace: my-proxy
+  name: simple
+spec:
+  infrastructure:
+    proxyContainer:
+      resources:
+        requests:
+          memory: '512Mi'
+        limits:
+          memory: '512Mi'
+----
+
+This is not a recommended configuration, but illustrates that your configuration is not combined with the default values in any way.
```

Comment:
unfortunately I think you have to illustrate this with a more dangerous configuration, like removing limits